libs/partners/pinecone/langchain_pinecone/vectorstores.py

```diff
@@ -200,6 +200,7 @@ def similarity_search_by_vector_with_score(
         k: int = 4,
         filter: Optional[dict] = None,
         namespace: Optional[str] = None,
+        include_id: Optional[bool] = False,
```

Comment:
We don't want to the search API right now.

Reply:
You might have missed a word in there. Are you suggesting not to include the 'include_id' argument? 

Is this related to you first comment which said that we need to add an 'id' attribute to Document? Which I guess would make the include_id argument unnecessary?